Translators/Incident/incident_tim_translator.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from pgquery import query_db
+from incident_tim_generator import get_action, get_effect, get_point, get_itis_codes, calculate_direction
+
+class IncidentFeature:
+    def __init__(self, properties, geometry):
+        self.id = properties["id"].replace("_", "-")
+        self.route_name = properties["routeName"].replace("_", "-")
+        self.problem = properties["type"]
+        self.additional_impacts = [] if properties.get("additionalImpacts") == None else properties["additionalImpacts"]
+        self.lane_impacts = properties["laneImpacts"]
+        self.traveler_information_message = properties["travelerInformationMessage"]
+        self.geometry = geometry["coordinates"]
+        self.geometry_type = geometry["type"]
+
+
+    def get_id(self):
+        return self.id
+    
+    def get_route_name(self):
+        return self.route_name
+    
+    def get_problem(self):
+        return self.problem
+    
+    def get_additional_impacts(self):
+        return self.additional_impacts
+    
+    def get_lane_impacts(self):
+        return self.lane_impacts
+    
+    def get_traveler_information_message(self):
+        return self.traveler_information_message
+    
+    def get_geometry(self):
+        return self.geometry
+    
+    def get_geometry_type(self):
+        return self.geometry_type
+    
+
+def translate(incident_geojson):
+    tims = {"timIncidentList": []}
```

Comment:
suggestion: docstring comments on our methods would be a nice touch for documentation going forward

Reply:
I added docstring comments on some of the larger methods and agree that this is a good plan for documentation going forward